ledger/acctupdates.go

```diff
@@ -212,11 +217,16 @@ func (au *accountUpdates) initialize(cfg config.Local, dbPathPrefix string, gene
 	if cfg.CatchpointFileHistoryLength < -1 {
 		au.catchpointFileHistoryLength = -1
 	}
+	// initialize the commitSyncerClosed with a closed channel ( since the commitSyncer go-routine is not active )
+	au.commitSyncerClosed = make(chan struct{})
+	close(au.commitSyncerClosed)
 }
 
 // loadFromDisk is the 2nd level initializtion, and is required before the accountUpdates becomes functional
 // The close function is expected to be call in pair with loadFromDisk
```

Comment:
What is meant by: "in pair with"? 
should close be called after or before loadFromDisk?

Reply:
"in pair with" means that every call to `loadFromDisk` should be followed ( eventually ) by a `close`, and not, for instance two subsequent calls to `loadFromDisk`